### PR TITLE
Added fixes to FE

### PR DESCRIPTION
**Pull request by oisinq**

### CODE REVIEW

File: src/main/java/universityWebApp/controller/HomeController.java [hunk 1]
```diff
@@ -46,15 +46,9 @@ public String viewHomePage(Model model) throws StudentNotFoundException, ModuleN
...
-            List<Module> modules = new ArrayList<>();
-
-            for (Long moduleId : coordinatedModules) {
-                modules.add(moduleRepository.findById(moduleId)
-                        .orElseThrow(() -> new ModuleNotFoundException(moduleId)));
-            }
+            List<Module> coordinatedModules = moduleRepository.findIDByCoordinator(staff.getId());
 
-            model.addAttribute("modules", modules);
+            model.addAttribute("modules", coordinatedModules);
```
Comment (DarraghClarke) [inline src/main/java/universityWebApp/controller/HomeController.java]: what if a module doesn't exist?

File: src/main/java/universityWebApp/controller/ModuleController.java [hunk 2]
```diff
@@ -107,7 +115,7 @@ public String editModule(@PathVariable("id") long moduleId, Model model) throws
      * This endpoint gets a specific module's details if it exists
      */
     @RequestMapping(value = "modules/{id}/edit", method = RequestMethod.POST)
-    public String editModule(ModelMap model, Module module) {
+    public String editModule(ModelMap model, Module module, @RequestParam long moduleId) {
```
Comment (DarraghClarke) [inline src/main/java/universityWebApp/controller/ModuleController.java]: should this be @PathParam

File: src/main/java/universityWebApp/controller/HomeController.java (hunk 1 above)
Comment (oisinq) [reply src/main/java/universityWebApp/controller/HomeController.java]: it just works

File: src/main/java/universityWebApp/controller/ModuleController.java (hunk 2 above)
Comment (oisinq) [reply src/main/java/universityWebApp/controller/ModuleController.java]: Yes

Comment (oisinq) [reply src/main/java/universityWebApp/controller/ModuleController.java]: But I don't use it, so I've thrown it out

File: src/main/java/universityWebApp/controller/HomeController.java (hunk 1 above)
Comment (DarraghClarke) [reply src/main/java/universityWebApp/controller/HomeController.java]: The code being removed had error catching though

Comment (oisinq) [reply src/main/java/universityWebApp/controller/HomeController.java]: Yes but there's no error to catch, if there are no modules, this still works. This code is one db read vs N reads, and there's no need for the error catching

**Pull request merged**